src/materials/SpriteMaterial.js

```diff
@@ -11,49 +11,50 @@ import { Color } from '../math/Color.js';
  * }
  */
 
-function SpriteMaterial( parameters ) {
+class SpriteMaterial extends Material {
 
-	Material.call( this );
+	constructor( parameters ) {
 
-	this.type = 'SpriteMaterial';
+		super();
 
-	this.color = new Color( 0xffffff );
+		this.type = 'SpriteMaterial';
 
-	this.map = null;
+		this.color = new Color( 0xffffff );
 
-	this.alphaMap = null;
+		this.map = null;
 
-	this.rotation = 0;
+		this.alphaMap = null;
 
-	this.sizeAttenuation = true;
+		this.rotation = 0;
 
-	this.transparent = true;
+		this.sizeAttenuation = true;
 
-	this.setValues( parameters );
+		this.transparent = true;
 
-}
+		this.setValues( parameters );
 
-SpriteMaterial.prototype = Object.create( Material.prototype );
-SpriteMaterial.prototype.constructor = SpriteMaterial;
-SpriteMaterial.prototype.isSpriteMaterial = true;
+	}
 
-SpriteMaterial.prototype.copy = function ( source ) {
+	copy( source ) {
 
-	Material.prototype.copy.call( this, source );
+		Material.prototype.copy.call( this, source );
 
-	this.color.copy( source.color );
+		this.color.copy( source.color );
 
-	this.map = source.map;
+		this.map = source.map;
 
-	this.alphaMap = source.alphaMap;
+		this.alphaMap = source.alphaMap;
 
-	this.rotation = source.rotation;
+		this.rotation = source.rotation;
 
-	this.sizeAttenuation = source.sizeAttenuation;
+		this.sizeAttenuation = source.sizeAttenuation;
 
-	return this;
+		return this;
 
-};
+	}
 
+}
+
+SpriteMaterial.prototype.isSpriteMaterial = true;
```

Comment:
These lines still needs to be changed, see https://github.com/mrdoob/three.js/pull/20100#discussion_r472421671.